services/user.py

```diff
@@ -0,0 +1,57 @@
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
+) -> User:
+    user = User()
+    user.username = username
+    user.set_password(password)
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
```

Comment:
Consider using `get_object_or_404` or handling the `DoesNotExist` exception to manage cases where a user with the given ID does not exist. This will prevent the application from crashing due to an unhandled exception.